src/s3fs.cpp

```diff
@@ -4540,14 +4545,33 @@ static int my_fuse_opt_proc(void* data, const char* arg, int key, struct fuse_ar
       passwd_file = strchr(arg, '=') + sizeof(char);
       return 0;
     }
+    if(0 == strcmp(arg, "ibm_auth")){
+      S3fsCurl::SetIsIBMAuth(true);
+      S3fsCurl::SetIAMCredentialsURL("https://iam.bluemix.net/oidc/token");
+      S3fsCurl::SetIAMTokenField("access_token");
+      S3fsCurl::SetIAMExpiryField("expiration");
+      S3fsCurl::SetIAMFieldCount(2);
+      // IBM auth only supports a-xmz-acl: public-read
+      if(S3fsCurl::SetDefaultAcl("") == "public-read"){
+        S3fsCurl::SetDefaultAcl("public-read");
```

Comment:
I don't understand this; does COS support `private` as well as `public-read`?

Reply:
My bad, it supports both private (implicitly as the default ACL) and public.
Will fix.